src/main/java/be/seeseemelk/mockbukkit/ServerMock.java

```diff
@@ -305,6 +305,15 @@ public PlayerMock getPlayer(int num)
 		return playerList.getPlayer(num);
 	}
 
+	/**
+	 * Returns the {@link MockPlayerList} instance that is used by this server.
+	 * @return The {@link MockPlayerList} instance.
+	 */
+	public MockPlayerList getPlayerList()
```

Comment:
Just noting the fact that this can be annotated with `@NotNull` if it is guaranteed to always return an instance of `MockPlayerList`